### ISSUE TITLE
Error Code after Load Savegame

### ISSUE THREAD

**Issue by kferrano** [Banjobeni/DerailValley-PersistentJobs#19]
```
Exception thrown at iterator CarSpawningJobGenerator.GenerateProceduralJobsCoroutine after yield at "initial":
System.TypeLoadException: Could not resolve type with token 010000cc (from typeref, class/assembly WaitFor, Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null)
  at PersistentJobsMod.Utilities.ExceptionCatchingCoroutineIterator.MoveNext () [0x00002] in <fa2c14d94b9649fab840fb5b15b255ba>:0 
```

**Comment by kferrano** [comment]
If I knew where your Discord link is

**Comment by T3M8029** [comment]
Link to Altfuture´s discord: [](https://discord.gg/sxVzQN6C).

**Comment by Banjobeni** [comment]
Fixed in 3.2.2.
